desktop/use-desktop/builds.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Explore Builds (Beta)
+description: Understand how to use the Builds view in Docker Desktop
+keywords: Docker Dashboard, manage, gui, dashboard, builders, builds
+---
+
+The **Builds**  view is a simple interface that lets you inspect your build
+history and manage builders using a graphical interface. By default, it
```

Comment:
🤔 the sentence starts with stating that it's an `interface` and here repeats it.